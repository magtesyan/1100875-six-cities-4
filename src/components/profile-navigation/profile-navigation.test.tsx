import * as React from "react";
import * as renderer from "react-test-renderer";
import ProfileNavigation from "./profile-navigation";

const isUserAuthorized = true;
const email = `fff@fff.ru`;

it(`Render Profile Navigation`, () => {
  const tree = renderer
    .create(
        <ProfileNavigation
          onSignInClick={jest.fn()}
          isUserAuthorized={isUserAuthorized}
          email={email}
          onEmailClick={jest.fn()}
          errorStatus={false}
        />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
