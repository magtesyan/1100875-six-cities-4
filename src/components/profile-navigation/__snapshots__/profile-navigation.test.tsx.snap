// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Profile Navigation 1`] = `
<nav
  className="header__nav"
>
  <div
    style={
      Object {
        "color": "red",
        "fontSize": "20px",
      }
    }
  >
    
  </div>
  <ul
    className="header__nav-list"
  >
    <li
      className="header__nav-item user"
      onClick={[Function]}
    >
      <a
        className="header__nav-link header__nav-link--profile"
        href="#"
      >
        <div
          className="header__avatar-wrapper user__avatar-wrapper"
        />
        <span
          className="header__user-name user__name"
        >
          fff@fff.ru
        </span>
      </a>
    </li>
  </ul>
</nav>
`;
